CollapsibleExample.xcodeproj/xcuserdata/lizzyrandall.xcuserdatad/xcschemes/CollapsibleExample.xcscheme

```diff
@@ -83,6 +83,11 @@
          </BuildableReference>
       </BuildableProductRunnable>
       <AdditionalOptions>
+         <AdditionalOption
```

Comment:
You might want to update your gitignore file so it doesn't include personal settings files. Here is MissionHub's https://github.com/CruGlobal/MissionHub-iOS/blob/master/.gitignore